src/java.desktop/macosx/classes/sun/lwawt/macosx/LWCToolkit.java

```diff
@@ -178,32 +166,28 @@ public ResourceBundle run() {
         }
     }
 
+    @SuppressWarnings("restricted")
+    private static void loadLibrary() {
+        System.loadLibrary("awt");
+        System.loadLibrary("fontmanager");
+    }
+
     /*
      * If true  we operate in normal mode and nested runloop is executed in JavaRunLoopMode
      * If false we operate in singleThreaded FX/AWT interop mode and nested loop uses NSDefaultRunLoopMode
      */
-    @SuppressWarnings("removal")
     private static final boolean inAWT
-            = AccessController.doPrivileged(new PrivilegedAction<Boolean>() {
-        @Override
-        public Boolean run() {
-            return !Boolean.parseBoolean(
+            = !Boolean.parseBoolean(
```

Comment:
You could have simplified this to use Boolean.getBoolean() but this is closer to the original so that's fine.